packages/cli/package.json

```diff
@@ -6,9 +6,7 @@
   "types": "dist/index.d.ts",
   "type": "module",
   "license": "MIT",
-  "bin": {
-    "mud": "./dist/mud.js"
-  },
+  "bin": "./dist/mud.js",
```

Comment:
This change is to avoid a binary conflict of `mud` coming from the cli package directly and the global `mud` package.